src/renderers/server-render.jsx

```diff
@@ -94,7 +93,10 @@ const getRouterRenderer = (store, baseUrl, clientFilename, assetPublicPath) =>
 				200;
 		} catch(e) {
 			// log the error stack here because Observable logs not great
-			console.error(e.stack);
+			console.error(JSON.stringify({
+				message: `Internal rendering error ${e.message}`,
```

Comment:
Looking into this, I realized I could just leave the logging to the server-level `onPreResponse` function, which has a reference to the `request`, rather than doing anything special here.